cluster/src/main/java/com/hartwig/pipeline/resource/Hg37ResourceFiles.java

```diff
@@ -0,0 +1,52 @@
+package com.hartwig.pipeline.resource;
+
+import static com.hartwig.pipeline.resource.ResourceNames.AMBER_PON;
+import static com.hartwig.pipeline.resource.ResourceNames.BEDS;
+import static com.hartwig.pipeline.resource.ResourceNames.GC_PROFILE;
+import static com.hartwig.pipeline.resource.ResourceNames.GRIDSS_REPEAT_MASKER_DB;
+import static com.hartwig.pipeline.resource.ResourceNames.MAPPABILITY;
+import static com.hartwig.pipeline.resource.ResourceNames.REFERENCE_GENOME;
+import static com.hartwig.pipeline.resource.ResourceNames.SAGE;
+import static com.hartwig.pipeline.resource.ResourceNames.SNPEFF;
+
+import com.hartwig.pipeline.execution.vm.VmDirectories;
+
+public class Hg37ResourceFiles implements ResourceFiles
+{
```

Comment:
Please apply the code style of the project. The project root directory contains hmf_code_schema.xml if you want to have it done automatically for this project.

Reply:
Changed